test/env.js

```diff
@@ -0,0 +1,18 @@
+const Environment = require('jest-environment-jsdom');
+const nodeCrypto = require('crypto');
+
+module.exports = class extends Environment {
+    async setup() {
+        await super.setup();
+        if (typeof this.global.TextEncoder === 'undefined') {
```

Comment:
Personally, I think a comment would be helpful to explain why this code is needed. For instance, why is the conditional statement necessary? Is it not known ahead of time whether or not we need to set these global values? I don't see `this.global` used elsewhere in the tests.

Reply:
`TextEncoder` and `TextDecoder` classes are needed by some iam-client-lib dependency and are provided by the browser environment, while in the Node these classes are contained in `util` package. Because of this I add them to globals. You are right that the purpose of this setup is unclear. I will add doc comments